amm/interp/src/main/scala/ammonite/interp/CompilerLifecycleManager.scala

```diff
@@ -145,7 +149,13 @@ class CompilerLifecycleManager(headFrame: => Frame){
     }
   }
 
-  def addToClasspath(classFiles: ClassFiles) = synchronized{
+  def preConfigureCompiler(callback: scala.tools.nsc.Settings => Unit) =
+    synchronized {
```

Comment:
wonder if this should be applied straight away ... looks like it doesn't get taken into account in the pre-processing of the next unit